packages/@webex/plugin-cc/src/types.ts

```diff
@@ -148,3 +148,24 @@ export interface SubscribeRequest {
 }
 
 export type EventResult = IAgentProfile;
+
+export const CHANNEL_NAME = {
+  TELEPHONY: 'telephony',
+  CHAT: 'chat',
+  SOCIAL: 'social',
+  EMAIL: 'email',
+} as const;
+
+export type CHANNEL_NAME = Enum<typeof CHANNEL_NAME>;
```

Comment:
We don't have to create a separate type like this. The constant defined above can be converted to an `enum`, no need to create a const variable.

Reply:
@rarajes2 , Can you please provide an example on how to do this?